src/Form/TranslationRequestForm.php

```diff
@@ -21,7 +63,7 @@ public function save(array $form, FormStateInterface $form_state) {
     $link = $entity->toLink($this->t('View'))->toRenderable();
 
     $message_arguments = ['%label' => $this->entity->label()];
-    $logger_arguments = $message_arguments + ['link' => render($link)];
+    $logger_arguments = $message_arguments + ['link' => $this->renderer->render($link)];
```

Comment:
This log is really not important. Better we don't render a link and keep the log simpler than to inject Renderer into aall the classes that extend this one. This was boilerplate/generated code.